IHP/IDE/SchemaDesigner/Parser.hs

```diff
@@ -187,8 +187,9 @@ sqlType :: Parser PostgresType
 sqlType = choice $ map optionalArray
         [ uuid
         , text
-        , bigint
+        , smallint
         , int
```

Comment:
The `"unexpected '8'" ` error likely happens here. The `int` parser is called before the `bigint` parser. `int` has a `try (symbol' "INT")`. Given an input string `INT8`, this `int` parser will consume the `INT` prefix and then is left with just `8`. Therefore the above error.

We can fix this by calling the `bigint` parser before the `int` parser.
Like:
```haskell
, smallint
, bigint
, int
```

Reply:
Of course, it was getting late last night. Fixed, with comment to help the next guy as well.

Reply:
(github didn't allow committing suggested change after I had fixed INT8 (outdated diff))
I like the suggestion of having less-used types at the end of the list, in this way we can cater to everyone but not confuse newbies.